tests/Http/Media/UploadMediaRequestHandlerTest.php

```diff
@@ -142,6 +142,29 @@ public function it_throws_if_description_is_missing(): void
         $this->uploadMediaRequestHandler->handle($request);
     }
 
+    /**
+     * @test
+     * @bugfix https://jira.uitdatabank.be/browse/III-5005
+     */
+    public function it_throws_if_no_a_file_was_uploaded_with_the_wrong_form_data_name(): void
```

Comment:
I think there is a small typo here and should be `it_throws_if_a_file_was_uploaded_with_the_wrong_form_data_name`